Libraries/Text/Text.js

```diff
@@ -32,6 +36,7 @@ const viewConfig = {
     ellipsizeMode: true,
     allowFontScaling: true,
     selectable: true,
+    selectionColor: ColorPropType,
```

Comment:
@janicduplessis you're right. I got confused by another place where there was proptype, most probably by mistake.